internal/clients/flag/flag.go

```diff
@@ -13,13 +21,27 @@ type Values struct {
 	Debug bool
 }
 
+func usage() {
+	fmt.Fprint(os.Stderr, "Usage: Nightfall DLP is used to scan content for sensitive information\n\n")
+	pflag.PrintDefaults()
+	os.Exit(2)
```

Comment:
Are exit status codes standardized in any way? Aside from `0` being a successful run

Reply:
hmm yeah why exit code 2 here? I would think we would want 0 for success

Reply:
Looking at git and ngrok they return a `0` code for their help command. I am unsure on the effects of returning different error codes. 

This `Usage` function is also called when the `pflag.Parse()` call fails to parse the flags from the command line, or there are unexpected flags.